ipinfo/handler.py

```diff
@@ -30,6 +25,11 @@
 )
 from . import handler_utils
 from .bogon import is_bogon
+from .continent import continents
+from .countries import countries
+from .currency import countries_currencies
+from .eu import eu_countries
+from .flags import countries_flags
```

Comment:
Let's just put all of this data into a single file with different arrays. You can call it `data.py`.